deploy/olm-catalog/dev/7.13.0-1/manifests/businessautomation-operator.clusterserviceversion.yaml

```
@@ -414,9 +416,9 @@ spec:
   maturity: dev
   provider:
     name: Red Hat
-  replaces: businessautomation-operator.7.12.1-2
+  #replaces: businessautomation-operator.7.12.1-1
```

probably just testing leftover?
```suggestion
  replaces: businessautomation-operator.7.12.1-1
```

---

oops, forget to remove it, thanls.